.gitignore

```diff
@@ -14,4 +14,7 @@ shard/res/meme.png
 tags
 .vs/*
 .vscode/*
+.idea/*
 www
+secret.yaml
+tilt_modules
```

Comment:
todo: remove this, it should not be ignored.

Also, todo: add tilt_modules to linguist-vendored

Reply:
done